backend/tracim_backend/lib/core/content.py

```diff
@@ -1435,6 +1440,8 @@ def update_content(self, item: Content, new_label: str, new_content: str=None) -
         return item
 
     def update_file_data(self, item: Content, new_filename: str, new_mimetype: str, new_content: bytes) -> Content:
+        if item.status and CONTENT_STATUS.get_one_by_slug(item.status).global_status == GlobalStatus.CLOSED.value:  # nopep8
```

Comment:
according to its naming, `CONTENT_STATUS` is expected to be a constant... not an object with methods. By the way, its definition is strange (the status "in progress" is not in its definition... why?)

Here the code is difficult to read. According to the use case, I expect something like this: 

```
if not item.get_content_status().is_editable():
    ...
```

or 
```
status = item.get_content_status()
if ContentStatus.is_editable(status):
    ...
```

Here, when reading the code, I want to understand that there are editable and readonly contents - I don't care how it is implemented.

By the way, it's not related to status only but also to archived/deleted status.

Reply:
Please encapsulate the `is_readonly` or `is_editable` definition in a function or method somewhere else